### PR TITLE
Math Magicians Components

### PR DESCRIPTION
**Pull request by samiullah997**
- I have deleted all the boilerplate from CRA (text, images, styles).
- I have created a directory called components.
- Inside components, I have created a new Calculator.js file.
- In Calculator.js, I have created a React component that matches the given design.
- I Used class-based components.

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @samiullah997 🙋‍♂️ , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your `README` file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present, so kindly update your README file 👌

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
```
Comment (roshan-bajgain) [inline src/App.js]: - [ ] According to the project requirement your app.js should be a class-based component. Please implement it

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <div className="row">
+            <div className="btn-zero">0</div>
+            <div>.</div>
+            <div className="orange">=</div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (roshan-bajgain) [inline src/components/calculator.js:47]: - [ ] In the upcoming project you will design the calculator on a small area as you have a `nav` bar and some content so please make the calculator visible in a small space. Your calculator is now covering the full height and width of the browser. Use the image below for improvement.
### Design
sssss

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <div className="orange">+</div>
+          </div>
+          <div className="row">
+            <div className="btn-zero">0</div>
+            <div>.</div>
+            <div className="orange">=</div>
+          </div>
+        </div>
+      </div>
+    );
```
Comment (roshan-bajgain) [inline src/components/calculator.js:45]: - [ ] Your calculator should match the design. Please stick with the design
- [ ] Please make improvements on the screen of the calculator
### Original design
ssssssssss
### Your design
your

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @samiullah997, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 😆 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @samiullah997,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**